lib/widgets/app.dart

```diff
@@ -48,22 +64,7 @@ class _PerAccountRootState extends State<PerAccountRoot> {
   @override
   Widget build(BuildContext context) {
     if (store == null) return const LoadingPage();
-    return PerAccountStoreWidget(
-        store: store!,
-        child: MaterialApp(
-          title: 'Zulip',
-          theme: ThemeData(
-              // This applies Material 3's color system to produce a palette of
-              // appropriately matching and contrasting colors for use in a UI.
-              // The Zulip brand color is a starting point, but doesn't end up as
-              // one that's directly used.  (After all, we didn't design it for that
-              // purpose; we designed a logo.)  See docs:
-              //   https://api.flutter.dev/flutter/material/ColorScheme/ColorScheme.fromSeed.html
-              // Or try this tool to see the whole palette:
-              //   https://m3.material.io/theme-builder#/custom
-              colorScheme: ColorScheme.fromSeed(seedColor: kZulipBrandColor)),
-          home: const HomePage(),
-        ));
+    return PerAccountStoreWidget(store: store!, child: widget.child);
   }
```

Comment:
```
widgets/app [nfc]: Pull UI out of data-management code
```

After this commit, there's still one more bit of UI being directed by `_PerAccountRootState`: the `LoadingPage()` that shows when `store == null`. Should we pull that out too somehow? I sense that you want to use `PerAccountRoot` in places where a "page" won't be the right thing to build. 🙂

Reply:
Good point, yeah. I had that thought but then forgot it. I'll leave a TODO.

Reply:
done